content/docs/started/k8s/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Kubeflow on Existing Kubernetes Clusters"
-description = "Instructions for installing Kubeflow on your existing clusters"
+title = "Kubernetes Install"
+description = "Instructions for installing Kubeflow on an existing Kubernetes clusters"
```

Comment:
Since "an" is singular, change "clusters" to "cluster".

Reply:
Good catch.